vscode-extension/package.json

```diff
@@ -70,8 +70,20 @@
       {
         "command": "relay.stopCompiler",
         "title": "Relay: Stop Compiler"
+      },
+      {
+        "command": "relay.createNewFragmentComponent",
+        "title": "Relay: Create New Fragment Component"
       }
     ],
+    "menus": {
+      "explorer/context": [
+        {
+          "command": "relay.createNewFragmentComponent",
+          "group": "Relay@1"
+        }
```

Comment:
Ideally when the extension has decided that it is in a relay project somehow, it would set an context variable so we can [use "when" to not show these](https://code.visualstudio.com/api/references/when-clause-contexts) unless useful